Packs/NetQuestOMX/Integrations/NetQuestOMX/NetQuestOMX.py

```diff
@@ -0,0 +1,327 @@
+                method="GET", url_suffix="/api/Systems/Filters/Address/Status/Optimization", headers=self.headers,
+            )
+        except Exception as e:
+            raise DemistoException(
+                "An error was occurred when optimizing the IP's list."
+            ) from e
+        return response
+
+    def address_list_create_request(self, name: str) -> requests.Response:
+        try:
+            response = self._http_request(
+                method="POST", url_suffix="/api/Systems/Filters/ListImport/Config/Install", headers=self.headers,
+                data={"Name": name}
+            )
+        except Exception as e:
+            raise DemistoException(
+                "An error was occurred when creating the IP's list."
+            ) from e
+        return response
+
+    def address_list_rename_request(self, new_name: str, existing_name: str) -> requests.Response:
+        try:
+            response = self._http_request(
+                method="PUT", url_suffix=f"/api/Systems/Filters/ListImport/ListName/{existing_name}/Config/Install",
+                headers=self.headers,
+                data={"Name": new_name}
+            )
+        except Exception as e:
+            raise DemistoException(
+                f"An error was occurred when renaming the {existing_name} IP's list to {new_name}."
+            ) from e
+        return response
+
+    def address_list_delete_request(self, list_name_to_delete: str) -> requests.Response:
+        try:
+            response = self._http_request(
+                method="DELETE",
+                url_suffix=f"/api/Systems/Filters/Address/ListName/{list_name_to_delete}/Config/List",
+                headers=self.headers,
+            )
+        except Exception as e:
+            raise DemistoException(
+                f"An error was occurred when deleting the {list_name_to_delete} IP's list."
+            ) from e
+        return response
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def address_list_upload_command(client: Client, args: dict):
+    """
+    This function uploads a .txt file with address list to the appliance.
+    The appliance temporarily stores the file until it is saved to the Library and replaces any previously loaded list file.
+
+    Returns:
+        A CommandResults containing a success indication or a DemistoException.
+    """
+    entry_id = args["entry_id"]  # a required argument
+    file_info = demisto.getFilePath(entry_id)
+    file_path = file_info['path']
+    file_name = file_info['name']
+
+    try:
+        shutil.copy(file_path, file_name)
+    except Exception as exc:
+        demisto.error(f'Failed to prepare file for upload. Error: {exc}')
+        raise DemistoException('Failed to prepare file for upload.') from exc
+
+    result = client.address_list_upload_request(file_name)
+
+    if result.status_code == '200':
+        return CommandResults(readable_output="Address list was successfully uploaded")
```

Comment:
Instead of doing this manually, pass `ok_codes=(200,)` to `_http_request`.
Same applies for all commands